arrow-core/src/main/kotlin/arrow/core/extensions/nonemptylist/applicative/NonEmptyListApplicative.kt

```diff
@@ -32,8 +32,8 @@ internal val applicative_singleton: NonEmptyListApplicative = object :
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-  "just()",
-  "arrow.core.just"
+    "NonEmptyList.just(this)",
+    "arrow.core.NonEmptyList"
```

Comment:
```suggestion
    "arrow.core.NonEmptyList", "arrow.core.just"
```

Reply:
Should we move towards `nonEmptyListOf` similar to the other collections constructors in Kotlin? cc\\ @raulraja